### PR TITLE
[R2] Add R2 Pricing page

### PR DESCRIPTION
**Pull request by kris-evans**

### CODE REVIEW

File: content/r2/platform/pricing.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+pcx-content-type: concept
+title: Pricing
+---
+
+# Pricing
+
+R2 charges based on the total volume of data stored, along with two classes of operations on that data - Class A operations, which are more expensive and tend to mutate state, and Class B operations, which tend to read existing state. There are no charges for egress bandwidth.
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
R2 charges based on the total volume of data stored, along with two classes of operations on that data – [Class A operations](#class-a-operations), which are more expensive and tend to mutate state, and [Class B operations](#class-b-operations) which tend to read existing state. There are no charges for egress bandwidth.
```

File: content/r2/platform/pricing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## R2
+
+{{<table-wrap>}}
+
+|                    | Forever Free                 | Paid - Rates                       | 
+| ------------------ | ---------------------------- | ---------------------------------- |
+| Storage            | 10 GB / month                | $0.015 / GB-month                  |
+| Class A Operations | 1,000,000 requests / month   | $4.50 / million requests           | 
+| Class B Operations | 10,000,000 requeests / month | $0.36 / million requests           |
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
| Class B Operations | 10,000,000 requests / month | $0.36 / million requests           | 
```

File: content/r2/platform/pricing.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### Storage Usage
+
+Storage for is billed using gigabyte-month (GB-month) and is a measure of how large your objects are and how long you store your
+objects for in the month.
+
+### Class A Operations
+
+The list of Class A Operations includes ListBuckets, PutBucket, ListObjects, PutObject, CopyObject, CompleteMultipartUpload, CreateMultipartUpload, UploadPart, 
+and UploadPartCopy.
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
Class A Operations include ListBuckets, PutBucket, ListObjects, PutObject, CopyObject, CompleteMultipartUpload, CreateMultipartUpload, UploadPart, and UploadPartCopy.
```

File: content/r2/platform/pricing.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### Class B Operations
+
+The list of Class B Operations includes HeadBucket, HeadObject, and GetObject.
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
Class B Operations include HeadBucket, HeadObject, and GetObject.
```

File: content/r2/platform/pricing.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+### Free Operations
+
+The list of free operations includes DeleteObject, DeleteBucket and DeleteMulitpartUpload.
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
Free operations include DeleteObject, DeleteBucket and DeleteMulitpartUpload.
```

File: content/r2/platform/pricing.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## R2 billing examples
+
+#### Example 1
+
+If a user writes 1000 1GB objects in R2 for 1 month and each object is requested 1000 times per month, the 
+estimated cost in a month would be:
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
If a user writes 1,000 objects in R2 for 1 month and each object is 1GB in size and requested 1,000 times per month, the estimated cost for the month would be:
```

File: content/r2/platform/pricing.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+Total = $14.85
+
+- (1,000,000 Class B Operations - included 10,000,000 Class B Operations) = $0.00
+- (1,000 Class A Operations - included 1,000,000 Class A Operations) = $0.00
+- (1000 GB-months - 10 included GB-months) = $14.85
```
Comment (lukeed) [inline content/r2/platform/pricing.md]: Trying a different format and spelling it out more:

```suggestion
{{<table-wrap>}}
|                    | Usage                                      | Free Tier    | Billable Quantity | Price      |
|--------------------|--------------------------------------------|--------------|-------------------|------------|
| Class B Operations | (1,000 objects) * (1,000 reads per object) |   10,000,000 |                 0 |      $0.00 |
| Class A Operations | (1,000 objects) * (1 write per object)     |    1,000,000 |                 0 |      $0.00 |
| Storage            | (1,000 objects) * (1GB per object)         | 10 GB-months |     990 GB-months |     $14.85 |
| **TOTAL**          |                                            |              |                   | **$14.85** |
{{</table-wrap>}}
```

File: content/r2/platform/pricing.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+#### Example 2
+
+If a user writes the same 1GB object 1,000,000 times a day and the object is read 10,000,000 times a day, 
+the stimated cost in a month would be:
```
Comment (lukeed) [inline content/r2/platform/pricing.md]:
```suggestion
If a user writes the same 1GB object 1,000,000 times a day and the object is read 10,000,000 times a day, the estimated cost in a month would be: 
```

File: content/r2/platform/pricing.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+#### Example 2
+
+If a user writes the same 1GB object 1,000,000 times a day and the object is read 10,000,000 times a day, 
+the stimated cost in a month would be: 
+
+Total = $234.50
+
+- (300,000,000 Class B Operations - included 10,000,000 Class B Operations) = $104.00
+- (30,000,000 Class A Operations - included 1,000,000 Class A Operations) = $130.50
+- (1 GB-months - 10 included GB-months) = $0.00
```
Comment (lukeed) [inline content/r2/platform/pricing.md]: The Class B Operations come to $104.40

```suggestion
{{<table-wrap>}}
|                    | Usage                                               | Free Tier    | Billable Quantity | Price       |
|--------------------|-----------------------------------------------------|--------------|-------------------|-------------|
| Class B Operations | (1 object) * (10,000,000 reads per day) * (30 days) |   10,000,000 |       290,000,000 |     $104.40 |
| Class A Operations | (1 object) * (1,000,000 writes per day) * (30 days) |    1,000,000 |        29,000,000 |     $130.50 |
| Storage            | (1 object) * (1GB per object)                       | 10 GB-months |       0 GB-months |        $0.00 |
| **TOTAL**          |                                                     |              |                   | **$234.90** |
{{</table-wrap>}}
```

File: content/r2/platform/pricing.md (hunk 1 above)
Comment (dpuga1) [reply content/r2/platform/pricing.md]: But who is getting that money not me

Comment (dpuga1) [reply content/r2/platform/pricing.md]: I am being completely used I don't know about any of this

Comment (dpuga1) [reply content/r2/platform/pricing.md]: Can somebody please help me

File: content/r2/platform/pricing.md (hunk 2 above)
Comment (dpuga1) [reply content/r2/platform/pricing.md]: How do I set that up to get paid

File: content/r2/platform/pricing.md (hunk 3 above)
Comment (dpuga1) [reply content/r2/platform/pricing.md]: Can u help me with that and I'll bring u on board .

Comment (dpuga1) [reply content/r2/platform/pricing.md]: I don't know what I'm doing so I really could use the help .

File: content/r2/platform/pricing.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+## R2
+
+{{<table-wrap>}}
+
+|                    | Forever Free                 | Paid - Rates                       | 
+| ------------------ | ---------------------------- | ---------------------------------- |
+| Storage            | 10 GB / month                | $0.015 / GB-month                  |
+| Class A Operations | 1,000,000 requests / month   | $4.50 / million requests           | 
+| Class B Operations | 10,000,000 requests / month | $0.36 / million requests           |
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
|                    | Forever Free                 | Paid - Rates                       | 
| ------------------ | ---------------------------- | ---------------------------------- |
| Storage            | 10 GB / month                | $0.015 / GB-month                  |
| Class A Operations | 1,000,000 requests / month   | $4.50 / million requests           | 
| Class B Operations | 10,000,000 requests / month  | $0.36 / million requests           | 
```

File: content/r2/platform/pricing.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
+---
+pcx-content-type: concept
+title: Pricing
+---
+
+# Pricing
+
+R2 charges based on the total volume of data stored, along with two classes of operations on that data – [Class A operations](#class-a-operations), which are more expensive and tend to mutate state, and [Class B operations](#class-b-operations) which tend to read existing state. There are no charges for egress bandwidth.
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
R2 charges based on the total volume of data stored, along with two classes of operations on that data:

 1. [Class A operations](#class-a-operations) which are more expensive and tend to mutate state. 
 2. [Class B operations](#class-b-operations) which tend to read existing state.
 
 There are no charges for egress bandwidth.
```

File: content/r2/platform/pricing.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+| Storage            | 10 GB / month                | $0.015 / GB-month                  |
+| Class A Operations | 1,000,000 requests / month   | $4.50 / million requests           | 
+| Class B Operations | 10,000,000 requests / month | $0.36 / million requests           | 
+
+{{</table-wrap>}}
+
+### Storage Usage
+
+Storage for is billed using gigabyte-month (GB-month) and is a measure of how large your objects are and how long you store your
+objects for in the month.
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
Storage for is billed using gigabyte-month (GB-month) and is a measure of how large your objects are and how long you store your objects for in the month.
```

File: content/r2/platform/pricing.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+### Class A Operations
+
+Class A Operations include ListBuckets, PutBucket, ListObjects, PutObject, CopyObject, CompleteMultipartUpload, CreateMultipartUpload, UploadPart, and UploadPartCopy.
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
Class A Operations include `ListBuckets`, `PutBucket`, `ListObjects`, `PutObject`, `CopyObject`, `CompleteMultipartUpload`, `CreateMultipartUpload`, `UploadPart`, and `UploadPartCopy`.
```

File: content/r2/platform/pricing.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+### Class B Operations
+
+Class B Operations include HeadBucket, HeadObject, and GetObject.
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
Class B Operations include `HeadBucket`, `HeadObject`, and `GetObject`.
```

File: content/r2/platform/pricing.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+### Free Operations
+
+Free operations include DeleteObject, DeleteBucket and DeleteMulitpartUpload.
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
Free operations include `DeleteObject`, `DeleteBucket` and `DeleteMulitpartUpload`.
```

File: content/r2/platform/pricing.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+## R2 billing examples
+
+#### Example 1
+
+If a user writes 1,000 objects in R2 for 1 month and each object is 1GB in size and requested 1,000 times per month, the estimated cost for the month would be:
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
If a user writes 1,000 objects in R2 for 1 month and each object is 1 GB in size and requested 1,000 times per month, the estimated cost for the month would be:
```

File: content/r2/platform/pricing.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+{{<table-wrap>}}
+|                    | Usage                                      | Free Tier    | Billable Quantity | Price      |
+|--------------------|--------------------------------------------|--------------|-------------------|------------|
+| Class B Operations | (1,000 objects) * (1,000 reads per object) |   10,000,000 |                 0 |      $0.00 |
+| Class A Operations | (1,000 objects) * (1 write per object)     |    1,000,000 |                 0 |      $0.00 |
+| Storage            | (1,000 objects) * (1GB per object)         | 10 GB-months |     990 GB-months |     $14.85 |
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md:56]: Is this supposed to signify 10 GB per month? If so, it should be written out as such.

File: content/r2/platform/pricing.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+| **TOTAL**          |                                            |              |                   | **$14.85** |
+{{</table-wrap>}}
+
+#### Example 2
+
+If a user writes the same 1GB object 1,000,000 times a day and the object is read 10,000,000 times a day, the estimated cost in a month would be:
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md]:
```suggestion
If a user writes the same 1 GB object 1,000,000 times a day and the object is read 10,000,000 times a day, the estimated cost in a month would be: 
```

File: content/r2/platform/pricing.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+#### Example 2
+
+If a user writes the same 1GB object 1,000,000 times a day and the object is read 10,000,000 times a day, the estimated cost in a month would be: 
+
+{{<table-wrap>}}
+|                    | Usage                                               | Free Tier    | Billable Quantity | Price       |
+|--------------------|-----------------------------------------------------|--------------|-------------------|-------------|
+| Class B Operations | (1 object) * (10,000,000 reads per day) * (30 days) |   10,000,000 |       290,000,000 |     $104.40 |
+| Class A Operations | (1 object) * (1,000,000 writes per day) * (30 days) |    1,000,000 |        29,000,000 |     $130.50 |
+| Storage            | (1 object) * (1GB per object)                       | 10 GB-months |       0 GB-months |        $0.00 |
```
Comment (deadlypants1973) [inline content/r2/platform/pricing.md:69]: Same comment as above.

**Review by deadlypants1973 — APPROVED** [review]
PCX review

File: content/r2/platform/pricing.md (hunk 19 above)
Comment (kris-evans) [reply content/r2/platform/pricing.md:69]: Same as above -- combination of storage + duration of storage.

**Pull request merged**